src/mondo_mapping_status.py

```diff
@@ -0,0 +1,115 @@
+"""Mapping status between Medgen and Mondo"""
```

Comment:
Repurposed some of my GARD code to create these attached atefacts.

Feels a bit duplicative, but this is probably fine for now. However I'm likely to do more of this for other ingests as well.
I did simplify some of it. Most of the code is the same between Medgen and GARD, but some things are special to each.

[medgen_terms_mapping_status.tsv.zip](https://github.com/monarch-initiative/medgen/files/12140721/medgen_terms_mapping_status.tsv.zip)
[obsoleted_medgen_terms_in_mondo.txt](https://github.com/monarch-initiative/medgen/files/12140722/obsoleted_medgen_terms_in_mondo.txt)